src/Provider/AbstractProvider.php

```diff
@@ -74,43 +74,21 @@ public function getHttpClient()
         return $client;
     }
 
-    /**
-     * Get the URL that this provider uses to begin authorization.
-     *
-     * @return string
-     */
+    // ProviderInterface
```

Comment:
Since these methods are in the interface, they shouldn't be in the abstract, unless we're actually implementing something, right? Let's remove them, unless you think there's a reason to keep them here.

Additionally, if are implementing them, we should use `{@inheritDoc}` in docblocks instead of this style of comment:

```
/**
 * {@inheritDoc}
 */
```